<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Login">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>UserName:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonLogin">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>311</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Don't have a acount?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonForgotg">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Forgot your password?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRequest">
     <property name="geometry">
      <rect>
       <x>188</x>
       <y>300</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Request a new one</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditUserNameLog">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>80</y>
       <width>311</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPasswordLog">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>311</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Register">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>UserName:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PhoneNumber:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditUserNameReg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>311</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPasswordReg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>311</width>
       <height>25</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPhoneNumberReg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>311</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRegister">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>311</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRegister_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>310</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Return Log</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="FindPassword">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>UserName:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PhoneNumber:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditUserNameFid">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>311</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPhoneNumberFid">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>170</y>
       <width>301</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSure">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>301</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Sure</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSure_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>300</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="NewPassword">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>New Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNewPassword">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSureNewPass">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Sure</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSureNewPass_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>280</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
